__tests__/SetsSelectors.test.js

```diff
@@ -820,6 +820,395 @@ describe('SetsSelectors', () => {
         });
     });
 
+describe('getBestAvgVelocityEver', () => {
+
+    test('get best avg velocity ever', () => {
+        const state = {
```

Comment:
- [ ] throw in some with different exercises in addition to rep length to prove it, specifically make it so if the exercise was bench, it would've been picked